.github/workflows/ci.yml

```
@@ -0,0 +1,315 @@
+          for i in {1..30}; do
+            if pg_isready -h localhost -p 5432 -U testuser; then
+              break
+            fi
+            echo "Waiting for PostgreSQL..."
+            sleep 2
+          done
+
+      - name: Run database migrations
+        env:
+          DATABASE_URL: postgres://testuser:testpass@localhost:5432/chatlogger_test?sslmode=disable
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:1:6: Double quote to prevent globbing and word splitting

(shellcheck)